panther_bringup/config/WH01.yaml

```diff
@@ -0,0 +1,8 @@
+wheel_radius: 0.1825
```

Comment:
I would use those parameters from panther_description

Reply:
As I stated in PR description, this is temporary. Panther_description for ros1 is still waiting for changes to be merged. This apply to other comments